src/Controllers/GagePageController.ts

```diff
@@ -1127,7 +1440,66 @@ module StreamStats.Controllers {
                 } else {
                     chart.yAxis[0].update({ type: 'linear' });
                 }
+            }; 
+            
+            
+        //checkbox to linear to log scale for discharge plot
+        public logScaleDischarge = false; // starts with it uncehcked
+            public toggleLogLinearDischarge () {
+                let chart = $('#chart3').highcharts();
+                if (this.logScaleDischarge) {
+                    chart.yAxis[0].update({ type: 'logarithmic' });
+                    chart.xAxis[0].update({ type: 'logarithmic' });
+                } else {
+                    chart.yAxis[0].update({ type: 'linear' });
+                    chart.xAxis[0].update({ type: 'linear' });
+                }
+            };   
+
+
+        //radio to show age in discharge plot
+        // public ageQualityData = true; // starts with it checked
+        public toggleData (dataType) {
```

Comment:
I wonder if we could call this toggleDischarge or just something so we know it is related to the discharge plot.